app/src/main/java/com/example/henzoshimada/feeltrip/EditMoodActivity.java

```diff
@@ -40,13 +55,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
         addItemsOnEmotionalStateSpinner();
         addItemsOnSocialSitualtionSpinner();
-        Button submitButton = (Button) findViewById(R.id.post_mood_button);
-        submitButton.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                //submitMood();
-            }
-        });
+        addListenerOnSubmitButton();
```

Comment:
Changed slight functionality of this Button listener, although it's largely the same as this deleted code.